server/internal/adapter/internalapi/client.go

```
@@ -0,0 +1,59 @@
+	conn, err := grpc.NewClient(host,
+		grpc.WithTransportCredentials(tlsCreds),
+		grpc.WithPerRPCCredentials(callCreds),
+	)
+	if err != nil {
+		log.Fatalf("failed to create gRPC client: %v", err)
+		return err
+	}
+	defer SafeClose(conn)
+
+	client := pb.NewReEarthVisualizerClient(conn)
+	handleFunc(client, ctx)
```

golangci-lint (1.64.8)

Error return value is not checked

(errcheck)